ocp4/profiles/coreos-ncp.profile

```diff
@@ -613,3 +613,13 @@ selections:
     #- audit_rules_etc_shadow_openat
     #- audit_rules_etc_shadow_open_by_handle_at
     #- directory_access_var_log_audit
+
+    # SC-28
+    #- encrypt_partitions
```

Comment:
Should CoreOS partitions be encrypted?

Reply:
I would very much welcome your opinion on whether this is required (especially coming from FedRAMP moderate perspective), but I think another question is also if it's possible at all. At least in FCOS it doesn't seem to be easily doable (https://github.com/coreos/fedora-coreos-tracker/issues/94). Not saying it's not doable at all, but it would help to know if this is a hard req so that we can optionally ask the RHCOS/FCOS teams to help us come up with something.

Reply:
Yes, this is required in SC-28, and SC-28(1). Partitions (even if root is only the partition) has to be encrypted.